lib/msf/core/db_manager/connection.rb

```diff
@@ -18,7 +18,7 @@ def after_establish_connection
       migrate
 
       # Set the default workspace
-      framework.db.workspace = framework.db.default_workspace
+      self.workspace = self.default_workspace
```

Comment:
removed infinite loop. fw.db -> initialized db manager which calls this method, which called fw.db (loop)